src/mlpack/bindings/markdown/print_docs.cpp

```diff
@@ -90,204 +139,237 @@ void PrintDocs(const std::string& bindingName,
   }
   cout << endl;
 
-  cout << doc.shortDescription << " ";
+  // Print short description for each language.
   for (size_t i = 0; i < languages.size(); ++i)
   {
-    cout << "[Detailed documentation](#" << languages[i] << "_"
-      << bindingName << "_detailed-documentation){: .language-detail-link #"
-      << languages[i] << " }";
+    BindingInfo::Language() = languages[i];
+    cout << "<div class=\"language-short-desc\" id=\"" << languages[i]
+        << "\" markdown=\"1\">" << endl;
+    if (addWrapperDocs[i])
+    {
+      string desc = replace_all_copy(
+          paramMethods[0].Doc().longDescription(), "|", "\\|");
+      cout << desc;
+    }
+    else
+    {
+      cout << doc.shortDescription << " ";
+      cout << "[Detailed documentation](#" << languages[i] << "_";
+      cout << bindingName << "_detailed-documentation){: .language-detail-link #"
+        << languages[i] << " }.";
```

Comment:
```suggestion
          << languages[i] << " }.";
```
Little style fix. :)